docs/api/README.md

```diff
@@ -83,13 +111,31 @@ If you do not care about exact dimensions, rather size class, it is better to us
 
 ```javascript
 // To make use of width and height in our component, we just access width and 
-// height as props:
-const Info = ({ width = 0, height = 0}) => (
-  <Text>{width}pt x {height}pt</Text>
+// height by using WithContainerDimensions helper:
+const Info = () => (
+  <WithContainerDimensions>{(width, height) => (<Text>{width}pt x {height}pt</Text>)}</WithContainerDimensions>
 );
 
-// then wrap component with withContainerDimensions
-const WrappedInfo = withContainerDimensions(Info);
+// when it is rendered inside Grid your component is provided with values.
+<Grid>
+// ...
+    <Info />
+// ...
+```
+
+### withContainerDimensions (Component) → Component
+
+HOC that offers same functionality as WithContainerDimensions. Provides current `width` and `height` that first outer grid uses to determine size, depending on grid's `relativeTo` prop it is either viewport or grid itself.
+
+- **width** - reference component width
+- **height** - reference component height
+
+If you do not care about exact dimensions, rather size class, it is better to use [withSizeClass](#withsizeclasscomponent--component) since it causes re-rendering only when class changes rather than when any of provided dimensions change.
+
+```javascript
+// To make use of width and height in our component, we just access width and 
+// height by using WithContainerDimensions helper:
+const WrappedInfo = withContainerDimensions((width, height)  => (<Text>{width}pt x {height}pt</Text>));
```

Comment:
This is our mistake, it was supposed to be split to be consistent with the rest of the code:
```jsx
const WrappedInfo = withContainerDimensions((width, height)  => (
  <Text>{width}pt x {height}pt</Text>
));
```